components/Menu.js

```diff
@@ -0,0 +1,20 @@
+const html = require('html-template-tag')
+
+const Menu = () => html`
+  <nav class="Menu-list">
+    <div class="Menu-box">
+      <a class="Menu-link" href="/">About</a>
+      <span style="padding: 0 6px;">&nbsp;|&nbsp;</span>
```

Comment:
Yikes! Can we rather use CSS? Did I put these in? :)

Reply:
Yeah, It was like that. I will clean up everything I can towards the end. Not to worry :wink: